### PR TITLE
Add home page copy seAP

### LINKED ISSUE #14: VVV/E Add copy for seAp home page
## Description 

Copy for home page needs to be added. Copy is here: https://docs.google.com/spreadsheets/d/1aFDhogqKhC2AogDuwdOVTlOytb3NxbsYjOaka43ssbw/edit?usp=sharing

## Plan

- [x] Get sign off from client
- [x] Add copy from Google Doc

## Acceptance Criteria

- [x] Copy matches Google Doc

### PR DESCRIPTION
**Pull request by AlinChindea**
Fixes #14 

## Description

- added copy from spreadsheet - [here](https://docs.google.com/spreadsheets/d/1aFDhogqKhC2AogDuwdOVTlOytb3NxbsYjOaka43ssbw/edit#gid=0)

@neontribe/ca-seap

### CODE REVIEW

File: src/Copy/BrandCopy.elm [hunk 1]
```diff
@@ -62,8 +62,8 @@ brandCopy key =
                 , icon = "✨"
```
Comment (katjam) [inline src/Copy/BrandCopy.elm]: phone

File: src/Copy/BrandCopy.elm [hunk 2]
```diff
@@ -62,8 +62,8 @@ brandCopy key =
...
                 , href = "✨"
```
Comment (katjam) [inline src/Copy/BrandCopy.elm]: needs a link and category and action

File: src/Copy/BrandCopy.elm [hunk 3]
```diff
@@ -84,51 +84,49 @@ brandCopy key =
...
+            CopyText "Accept"
 
         CookieDecline ->
-            CopyText "✨"
+            CopyText "Decline"
 
         --Home Page
         HomeReadAboutH2 ->
-            CopyText "✨"
+            CopyText "When you are ill you need to be treated well. Find out how we can help you  speak up when things go wrong."
```
Comment (katjam) [inline src/Copy/BrandCopy.elm]: extra space between speak and you

File: src/Copy/BrandCopy.elm [hunk 4]
```diff
@@ -84,51 +84,49 @@ brandCopy key =
...
 
         HomeBannerImageAlt ->
-            CopyText "✨"
+            CopyText ""
 
         HomeTalkToSomeoneH2 ->
-            CopyText "✨"
+            CopyText "Your rights, your voice, your choice  "
```
Comment (katjam) [inline src/Copy/BrandCopy.elm]: Extra trailing whitespace

File: src/Copy/BrandCopy.elm [hunk 5]
```diff
@@ -84,51 +84,49 @@ brandCopy key =
...
 
         HomeP1 ->
-            CopyWithLink
-                { textBefore = "✨"
-                , linkText = "✨"
-                , destination = "#"
-                , textAfter = ""
-                }
+            CopySection
+            [ CopyText "When something goes wrong and you feel like the NHS has let you down it can be hard. You have a right to complain and the NHS needs your feedback in order to improve their service.  Call seAp for free information about getting your voice heard and other organisations that can help."
```
Comment (katjam) [inline src/Copy/BrandCopy.elm]: I think there is an extra space at the end of sentence 2

File: src/Copy/BrandCopy.elm [hunk 6]
```diff
@@ -84,51 +84,49 @@ brandCopy key =
...
+            , CopyText "If there are parts of your complaint you cannot do for yourself, you can ask for one of the seAp team to be your advocate. They will listen to you, support you to speak for yourself, and help you to make choices. They can also write letters for you, help you in complaint meetings, and talk to other people for you if necessary."
+            ]
 
         HomeP2 ->
-            CopyText "✨"
+            CopyText "The seAp service is completely confidential, free and independent of the NHS.  It can only be used to complain about NHS services, we can’t help with complaints about private health care or social care provided by the local council."
```
Comment (katjam) [inline src/Copy/BrandCopy.elm]: Extra space after NHS. ?

**Review by katjam — CHANGES_REQUESTED** [review]
Call to action meta missing + a few stray spaces.

**Comment by AlinChindea** [comment]
@katjam good spotting! :eyes: :fishing_pole_and_fish:

**Review by katjam — APPROVED** [review]
Looking good! :sake:

**Pull request merged**